src/main/java/it/gov/pagopa/pu/workflow/wf/classification/assessments/CreateAssessmentsWFClient.java

```diff
@@ -0,0 +1,47 @@
+package it.gov.pagopa.pu.workflow.wf.classification.assessments;
```

Comment:
it's not really related to classifications WF
```suggestion
package it.gov.pagopa.pu.workflow.wf.assessments;
```